### ISSUE TITLE
ignoreDelete from a device

### ISSUE THREAD

**Issue by erdnaxeli** [syncthing/syncthing#5040]
Hi,

A feature that I would like to see, is `ignoreDelete` but from a whole device, for every folder shared with it.

An example is the use of smartphones. Those devices have limited devices, so I may delete files to free some space. I should use `.stignore` for this, but I want to protect myself from any error I could make. I want to say: "this device can not be trusted, ignore all deletes from it".

Thanks.

**Comment by erdnaxeli** [comment]
I agree but what if I have many shares with this devices ? I could have
only one setting.

**Comment by AudriusButkevicius** [comment]
You do the same thing for many shares.
